docs/pages/custom-builds/schema.mdx

```diff
@@ -1300,6 +1300,98 @@ build:
   href="https://github.com/expo/eas-build/blob/main/packages/build-tools/src/steps/functions/uploadArtifact.ts"
 />
 
+#### `eas/install_maestro`
+
+Makes sure [Maestro](https://maestro.mobile.dev), the mobile UI testing framework, is installed, along with all its dependencies. 
+
+```yaml build-and-test.yml
+build:
+  name: Build and test
+  steps:
+    - eas/build
+    # ... simulator / emulator setup
```

Comment:
```suggestion
    ... simulator/emulator setup
```

- Is it a requirement to mention emulator/simulator setup? I assume the following maestro job won't work with out it.

Reply:
What do you mean?

> I assume the following maestro job won't work with out it.

Indeed, without simulator/emulator set up `maestro test` will fail.

Reply:
I had an idea that if we don't mention this step in the example snippet that maestro test might now work. Just wanted to confirm. Thank you!